### DISCUSSION TITLE
icons for html and css

### DISCUSSION THREAD

**Discussion by SireTallest** [tailwindlabs/heroicons#1075] [Ideas]
can i get social media icons for html/css?

**Comment by allypetroova** [comment]
Icons can be found on various resources.
They must be in SVG or PNG format, you can also use Font Awesome Icons (for example - facebook)

The first option:
`<a href="#" class="fa fa-facebook"></a>`

The second option ( `<head>` section):
`<link rel=”stylesheet” href=”https://link”>`
